src/main/java/cam72cam/mod/world/World.java

```diff
@@ -115,10 +115,12 @@ private void checkLoadedEntities() {
                     this.onEntityAdded(entity);
                 }
             }
-            for (Entity entity : new ArrayList<>(this.entityByID.values())) {
-                if (!this.internal.loadedEntityList.contains(entity.internal)) {
-                    ModCore.warn("Dropping entity that was not removed correctly %s - %s", entity.getUUID(), entity);
```

Comment:
Did you drop this log on purpose?

Reply:
Ah, no. Let me throw it back in.

Reply:
I've added it back in. Apologies for the oversight.